stage1/tasks/portfolio/portfolio-part-2.md

```diff
@@ -0,0 +1,106 @@
+
+## Criteria for Evaluation
+
+**Maximum score for the task: 130 points**
+
+The scoring for each requirement is detailed in the [CrossCheck Criteria](#crosscheck-criteria-130-points) section.
+
+Any disputed issues should be resolved in favor of the student being assessed.
+
+### CrossCheck Criteria (130 points)
+
+1. Implementation of the **burger menu**: **+40**
+   - At 768px and below, the navigation panel hides, and a burger icon appears: **+4**
+   - The burger icon is created using HTML and CSS without the use of images/svg: **+4**
+   - When clicking the burger icon, the burger menu slides out from the right, and the burger icon smoothly transforms into a cross: **+4**
+   - The burger menu occupies the entire available screen area below the `<header>` block: **+4**
+   - The placement and sizes of elements in the burger menu correspond to the layout (horizontal and vertical centering of menu items): **+4**
+   - When clicking the cross, the burger menu smoothly hides, moving to the right of the screen, and the cross smoothly transforms into a burger icon: **+4**
+   - When clicking on any link in the menu, the burger menu smoothly hides to the right, and the cross smoothly transforms into a burger icon: **+4**
+   - Links scroll smoothly to their anchor points: **+4**
+   - The page behind the open menu does not scroll. After closing the menu, scrolling works again: **+4**
+   - Above 768px, both the burger icon and menu are hidden, and the standard navigation panel is shown: **+4**
+2. Implementation of the **Slider**: **+40**
+
+- By default, the slider track must be horizontally centered on the page: **+4**
+- On desktop, slider scrolling is activated by hovering the cursor over the left or right area: **+8**
+- On desktop, the active area for scrolling the slider is ~30% of the screen width on each side: **+8**
+- On desktop, the remaining area in the center is inactive: **+4**
+- On mobile, the slider is scrolled by swiping with a finger: **+8**
+- The slider's end positions are aligned with the page content on either side. Scrolling is locked beyond these points: **+8**
+
+3. Implementation of the **accordions** in the `FAQ` section: **+24**
+
+- By default, the first accordion is open: **+4**
+- You can open or close the accordion by clicking anywhere on its header: **+4**
+- Only one accordion can be open at a time (opening a new one closes the previous): **+8**
+- Accordion state persists after page reload: **+8**
+
+4. Implementation of the **modals**: **+22**
+
+- A modal opens when you click any "BOOK NOW" button in the cards within the `Packages & Pricing` section: **+4**
+- The part of the page outside the Modal is darkened: **+4**
+- When the Modal is open, the vertical scroll of the page becomes inactive; when closed, it becomes active again: **+6**
```

Comment:
Maybe it's unfair that we're giving 6 points for a part that's similar to the one about blocking scrolling when opening the burger menu. It might be better to give 6 points to `A modal opens when you click any "BOOK NOW" button in the cards within the Packages & Pricing section: **+6**`